skypy/galaxies/velocity_dispersion.py

```diff
@@ -0,0 +1,66 @@
+r"""Models of galaxy velocity dispersion.
+
+"""
+
+import numpy as np
+import scipy.special as special
+
+__all__ = [
+    'schechter_vdf',
+]
+
+def schechter_vdf(vd_min, vd_max, size=None, resolution=1000):
+    r"""Sample velocity dispersion of elliptical galaxies in the local universe following a Schecter function.
+
+    Parameters
+    ----------
+    vd_min, vd_max: int
+        Lower and upper bounds of random variable x. Samples are drawn uniformly from bounds.
+    resolution: int
+        Resolution of the inverse transform sampling spline. Default is 100.
+    size: int
+        Number of samples returned. Default is 1.
+
+    Returns
+    -------
+    velocity_dispersion: array_like
+        Velocity dispersion drawn from Schechter function.
+
+    Notes
+    -----
+    The probability distribution function :math:`p(\sigma)` for velocity dispersion :math:`\sigma`
+    can be described by a Schechter function (see eq. (4) in [1]_)
+
+    .. math::
+
+        {dn}={\phi }_{*}{(\frac{\sigma}{{\sigma}_{*}})}^{\alpha}
+        {exp}[-{(\frac{\sigma}{{\sigma}_{*}})}^{\beta}]
+        \frac{\beta}{{\Gamma}(\alpha/\beta)}\frac{1}{\sigma},
```

Comment:
```suggestion
        \mathrm{d}\phi(\sigma) = \phi _{*} \left(\frac{\sigma}{\sigma_{*}}\right)^{\alpha}
        \exp\left[- \left(\frac{\sigma}{\sigma_{*}}\right)^{\beta}\right]
        \frac{\beta}{\Sigma(\alpha/\beta)}\frac{1}{\sigma} ,
        
     where :math:`\Sigma` is ... , :math:`\sigma_*` is ... , :math:`\phi_*` is ... and :math:`\alpha` and :math:`\beta` are free parameters.
```
Fill the gaps please.

Reply:
Thanks for the added explanation. Could you maybe also clean up the LaTeX code a bit? I.e. removing unnecessary curly brackets etc and please keep consistent with the equation from the paper. Instead `dn` write `phi = ...` and use `Sigma` instead of `Gamma`. See my suggestions.

Reply:
I am a bit confused by what Sigma is. In the paper there is no mention of it. However in this paper: https://iopscience.iop.org/article/10.1088/0004-637X/811/1/20/pdf, equation 3 uses the modified schechter function and  uses the Gamma function. What should I add as to what Sigma is?

Reply:
Hmm ok. In this case I guess it is the gamma function. I think it better to use `Gamma` but still mention that it is the gamma function.

Reply:
Got it.